opdracht-1.js

```diff
@@ -0,0 +1,33 @@
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+const tvNames = inventory.map((item)=>{
+    return item.name
+})
+console.log('Opdracht 1a')
+console.log(tvNames)
+
+// Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const soldOut = inventory.filter((item)=>{
+    return item.originalStock - item.sold < 1
+})
+console.log('Opdracht 1b')
+console.log(soldOut)
+
+// Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+
+const  ambiLight = inventory.filter((item)=>{
+    return item.options.ambiLight === true
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return item.options.ambiLight;
```